pkg/reconciler/nscert/nscert_test.go

```diff
@@ -623,7 +623,7 @@ func networkConfig() *netcfg.Config {
 func domainConfig() *routecfg.Domain {
 	domainConfig := &routecfg.Domain{
 		Domains: map[string]*routecfg.LabelSelector{
-			"example.com": {},
+			"svc.cluster.local": {},
```

Comment:
I think we should keep `example.com` for this nscert test.
net-certmanager does not create certificates for cluster local (`svc.cluster.local`) so we should assume that users set the domain as `example.com` as it is.

Reply:
sounds good... in that vein, I added an overlay to set the domain for some of the e2e tests, let me know if you think we should remove that